### PR TITLE
Adapting to new parent POM

### PR DESCRIPTION
**Pull request by recena**
@reviewbybees, specially @andresrc

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -35,6 +39,13 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>git</artifactId>
             <version>2.3</version>
+            <exclusions>
+                <!-- To avoid ClassNotFoundException during InjectedTest -->
+                <exclusion>
+                    <groupId>org.apache.httpcomponents</groupId>
+                    <artifactId>httpclient</artifactId>
+                </exclusion>
+            </exclusions>
```
Comment (tfennelly) [inline pom.xml]: Needed for upgrading pom?

File: src/main/java/org/jenkinsci/plugins/github_branch_source/PullRequestGHEventSubscriber.java [hunk 2]
```diff
@@ -59,11 +59,13 @@
...
-            for (SCMSource source : owner.getSCMSources()) {
-                if (source instanceof GitHubSCMSource) {
-                    return true;
+        if (project != null) {
+            if (project.getParent() instanceof SCMSourceOwner) {
+                SCMSourceOwner owner = (SCMSourceOwner) project.getParent();
+                for (SCMSource source : owner.getSCMSources()) {
+                    if (source instanceof GitHubSCMSource) {
+                        return true;
+                    }
```
Comment (tfennelly) [inline src/main/java/org/jenkinsci/plugins/github_branch_source/PullRequestGHEventSubscriber.java]: Needed for upgrading pom?

Comment (recena) [inline src/main/java/org/jenkinsci/plugins/github_branch_source/PullRequestGHEventSubscriber.java]: Yes. I need it. This new parent pom includes [Findbugs 3.0.3](https://github.com/jenkinsci/plugin-pom/blob/master/pom.xml#L328) and this version detects an issue.

File: pom.xml (hunk 1 above)
Comment (recena) [inline pom.xml]: Yes, I need it because `git` plugin includes a old version of `httpclient` and ATH needs **its** version.

Comment (recena) [inline pom.xml]: It seems @jglick had to do the same [here](https://github.com/jenkinsci/workflow-plugin/blob/master/aggregator/pom.xml#L102).

**Comment by amuniz** [comment]
:bee:

File: pom.xml (hunk 1 above)
Comment (tfennelly) [inline pom.xml]: Perfect, thanks.

File: src/main/java/org/jenkinsci/plugins/github_branch_source/PullRequestGHEventSubscriber.java (hunk 2 above)
Comment (tfennelly) [inline src/main/java/org/jenkinsci/plugins/github_branch_source/PullRequestGHEventSubscriber.java]: Perfect, thanks.

**Comment by tfennelly** [comment]
:bee:

**Comment by recena** [comment]
@amuniz @tfennelly Thanks!

**Comment by recena** [comment]
@reviewbybees done

**Comment by andresrc** [comment]
"retroactive" :bee:

**Pull request merged**